contracts/cw-dao/src/contract.rs

```diff
@@ -321,6 +331,7 @@ pub fn execute_update_config(
     max_voting_period: Duration,
     proposal_deposit_amount: Uint128,
     proposal_deposit_token_address: String,
+    refund_failed_proposals: Option<bool>,
```

Comment:
Not your fault, but Clippy isn't going to let this pass because this function has too many arguments. We should make it take one argument, an `UpdateContfigMsg`.

Would look like so:
```rust
pub fn execute_update_config(
    deps: DepsMut,
    env: Env,
    info: MessageInfo,
    config: UpdateConfigMsg,
) -> Result<Response<Empty>, ContractError> {
    ...
}
```

`UpdateConfigMsg`:
```rust
pub struct UpdateConfigMsg {
        threshold: Threshold,
        max_voting_period: Duration,
        proposal_deposit_amount: Uint128,
        proposal_deposit_token_address: String,
        refund_failed_proposals: Option<bool>,
}
```